components/rule/index.css

```diff
@@ -12,47 +12,54 @@ governing permissions and limitations under the License.
 
 @import '../commons/index.css';
 
+:root {
+  --spectrum-divider-vertical-height: 100%;
+
+  /* Work around DNA misspelling */
+  --spectrum-divider-small-vertical-width: var(--spectrum-divider-small-vertical-width, var(--spectrum-divider-small-veritcal-width));
```

Comment:
The current version of DNA has this token, but it's spelled wrong `var(--spectrum-divider-small-veritcal-width)`. This works around that issue, and should work properly once the misspelled token is removed.